src/structures/Node.ts

```diff
@@ -1,120 +1,135 @@
+import WebSocket from "ws";
+import fetch, { Response } from "node-fetch";
 import { Manager } from "./Manager";
+import { Player } from "./Player";
+import { ChiroError, ChiroEventError, ChiroEventErrorKind } from "./Error";
 import { NodeOptions, Payload, TrackData } from "../Static/Interfaces";
-
-import WebSocket from "ws";
-import fetch from "node-fetch";
 import { WSEvents, WSOpCodes } from "../Static/Constants";
-import { Player } from "./Player";
+
+function generateRandomPassword(): string {
```

Comment:
how about

```ts
import { randomBytes } from "crypto";

function generateRandomPassword(): string {
    return randomBytes(11).toString("hex");
}
```

Reply:
Btw just now rohnak said the password has to be set from Nexus server so there is no need of a random password generation. Is it ok to keep something like this?

Reply:
Yes, password needs to be specified on the server so it's a good idea to remove this

Reply:
removing this since the password is static on nexus and it can't be set from client side. better to remove it and put default password as SwagLordNitroUser12345